src/Compatibility/Microsoft.DotNet.PackageValidation/GetLastStablePackage.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Microsoft.Build.Framework;
+using Microsoft.NET.Build.Tasks;
+using NuGet.Common;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    public class GetLastStablePackage : TaskBase
+    {
+        [Required]
+        public string PackageId { get; set; }
+
+        public string[] NugetFeeds { get; set; }
+
+        public string LocalPackagesPath { get; set; }
+
+        public bool UseLocalPackagesPath { get; set; }
```

Comment:
It seems there is only a single place this task is called and there that boolean is always true. Why make this conditional?